ramls/loan.json

```
@@ -93,6 +93,11 @@
       "description": "Indicates whether or not this loan had its due date modified by a recall on the loaned item",
       "type": "boolean"
     },
+    "declaredLostDate" : {
+      "description": "Date and time loan was declared lost",
```

I think it could be good to get wording from Emma to describe this. I don't think the loan is declared lost.

```suggestion
      "description": "Date and time the item was declared lost during this loan",
```

---

Done.

---

Indeed, the loan is not declared lost, it's the item.